src/components/Publish/Pricing/Fees.tsx

```diff
@@ -37,6 +46,14 @@ export default function Fees({
   pricingType: 'dynamic' | 'fixed'
 }): ReactElement {
   const [field, meta] = useField('pricing.swapFee')
+  const [opcFees, setOpcFees] = useState<OpcFeesData>(undefined)
+  const { chainId } = useWeb3()
+
+  useEffect(() => {
+    getOpcFeees(chainId).then((response: OpcFeesData) => {
```

Comment:
without a wallet, `chainId` is not present leading to errors, so falling back to `1` should be more safe in that case: `chainId || 1`